src/System.Management.Automation/engine/CommandSearcher.cs

```diff
@@ -1010,24 +1010,23 @@ private CmdletInfo GetNextCmdlet()
                         return null;
```

Comment:
@powercode, your last commit had 28 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Validate that get-help works for AllUsers Scope.for module : Microsoft.PowerShell.Core.Validate -Description and -Examples sections in help content. Run 'Get-help -name <cmdletName>
```powershell
Cannot validate argument on parameter 'Name'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\HelpSystem.Tests.ps1: line 138
138:             $help = Get-Help -Name $cmdletName

```
Verify approved aliases list.All approved Cmdlets present (no new Cmdlets added, no Cmdlets removed)
```powershell
Expected $null or empty, but got @('Add-History', 'Clear-History', 'Connect-PSSession', 'Debug-Job', 'Disable-ExperimentalFeature', 'Disable-PSRemoting', 'Disable-PSSessionConfiguration', 'Disconnect-PSSession', 'Enable-ExperimentalFeature', 'Enable-PSRemoting').
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\DefaultCommands.Tests.ps1: line 584
584:         $result | Should -BeNullOrEmpty
```
Verify approved aliases list.All present Cmdlets should have the correct ConfirmImpact
```powershell
Expected $null or empty, but got @(@{Name=New-Variable; ConfirmImpact=Low; SideIndicator==>}, @{Name=Receive-PSSession; ConfirmImpact=Low; SideIndicator==>}, @{Name=Set-Location; ConfirmImpact=None; SideIndicator==>}, @{Name=Set-MarkdownOption; ConfirmImpact=None; SideIndicator==>}, @{Name=Set-PSBreakpoint; ConfirmImpact=None; SideIndicator==>}, @{Name=Set-PSDebug; ConfirmImpact=None; SideIndicator==>}, @{Name=Set-PSSessionConfiguration; ConfirmImpact=Medium; SideIndicator==>}, @{Name=Set-Service; ConfirmImpact=Medium; SideIndicator==>}, @{Name=Set-StrictMode; ConfirmImpact=None; SideIndicator==>}, @{Name=Set-TimeZone; ConfirmImpact=Medium; SideIndicator==>}).
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\DefaultCommands.Tests.ps1: line 606
606:         $result | Should -BeNullOrEmpty
```
File encoding tests.ParameterType for parameter 'Encoding' should be 'Encoding'.Encoding parameter of command '<Command>' is type 'Encoding'
```powershell
Cannot index into a null array.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\Encoding.Tests.ps1: line 14
14:             $command.Parameters['Encoding'].ParameterType.FullName | Should -BeExactly "System.Text.Encoding"

```
Implicit remoting on restricted ISS.restrictions works.Only 9 commands are public
```powershell
Expected 9, but got 8.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Implicit.Remoting.Tests.ps1: line 1497
1497:                 @(Invoke-Command $session { Get-Command }).Count | Should -Be 9
```